content/release-10-18-0/whatsnew-10-18-0.md

```diff
@@ -7,6 +7,39 @@ layout: bundle
 Release 10.18.0 includes the following new features or major feature enhancements.
 
 ### Application enablement
+#### Global Time Selector for dashboards
```

Comment:
Please add a link to the documentation where this is described.

Reply:
added, but since this wasn't described, the link won't work yet. I've create a PR in the c8y-docs repo..

Reply:
I have done some more changes, especially to the phrasing of the links and approved. @eniosultan Have the changes for the "Extensions" page been done in the user guide?

Reply:
So extensions info to the users-guide was added, but we haven't replaced entirely packages within the docs as it is still a correct term. For example the Package page was renamed to Extensions: 
![image](https://github.com/SoftwareAG/c8y-releasenotes/assets/92023325/0aa87bd7-de30-41d9-af8b-c554ee48837b)

But when adding a new one, it's called extension package --> 
![image](https://github.com/SoftwareAG/c8y-releasenotes/assets/92023325/7d9f9227-aa07-48f2-9abb-1c421c54a015)

Reply:
Ok, sounds good.

Reply:
Can you please approve the PR then?

Reply:
approved